frontend/components/Course/CourseWrapper.tsx

```diff
@@ -101,6 +101,7 @@ const CourseWrapper = ({ render, ...props }: CourseProps) => {
                                                     {notifs ? "ON" : "OFF"}
                                                 </div>
                                             }
+                                            size="mini"
                                         >
                                             <p>
                                                 Browser permissions are{" "}
```

Comment:
I would just not display anything regarding permissions if Notifications is not in window.